pallets/marketplace/src/lib.rs

```diff
@@ -744,6 +744,7 @@ pub mod pallet {
 
             match requirements.instant_match {
                 Some(sources) => {
+                    // ignore remaining rewards; do not pay out the matcher which is the same as the one registering
```

Comment:
This comment does not say much in this location

Reply:
we are ignoring the result here, made it more clear with `let _ =`